index.html

```diff
@@ -261,10 +261,10 @@ <h1>Nextcloud Desktop Client<a class="headerlink" href="#desktop" title="Permali
 								<p>Once you have a Nextcloud Server running, you can connect to it with various <a class="reference external" href="https://nextcloud.com/install/#install-clients">clients like our mobile and desktop client.</a>
 								Find documentation for the desktop client below.</p>
 								<div class="section" id="desktop-31">
-									<h2>Desktop 3.1<a class="headerlink" href="#desktop-31" title="Permalink to this headline">¶</a></h2>
+									<h2>Desktop 3.2<a class="headerlink" href="#desktop-31" title="Permalink to this headline">¶</a></h2>
```

Comment:
You should append the link to the 3.2 documentation instead of replacing the 3.1 one.

Reply:
@mgallien Yes. My bad. Fixed.